graph/telemetry/istio/appender/service_entry.go

```diff
@@ -225,3 +231,22 @@ func (a ServiceEntryAppender) getServiceEntry(serviceName string, globalInfo *gr
 
 	return nil, false
 }
+
+func isExportedToNamespace(se *serviceEntry, namespaceInfo *graph.AppenderNamespaceInfo) bool {
+	if se.exportTo == nil {
+		return true
+	}
+	for _, export := range se.exportTo.([]interface{}) {
+		if export == "*" {
+			return true
+		}
+		if export == "." && se.namespace == namespaceInfo.Namespace {
+			return true
+		}
+		if export == se.namespace {
```

Comment:
I think this should be commented out, or removed, because according to the documentation, it is not implemented yet in istio: https://istio.io/latest/docs/reference/config/networking/service-entry/
Says:

```
NOTE: in the current release, the exportTo value is restricted to “.” or “*” (i.e., the current namespace or all namespaces).
```

That note exists since v1.0 at least (i didn't dig further), so I guess we shouldn't expect the feature to be implemented any time soon ?

Reply:
@jotak I didn't actually see that comment, I had looked at the initial description and the fact it allowed a list of values.  Despite the current restriction I'm inclined to leave this code here as it shouldn't harm anything and will cover the use case if and when it ever shows up in Istio.  Are you OK with that?

Reply:
yes, sounds good!

Reply:
Great, thanks.  If you have no other comments feel free to approve :)